it/mail/chrome/messenger/messenger.properties

```diff
@@ -623,115 +623,115 @@ twitter.authorization.dialog.height=480
+photobucketservice.uploadProgress.text=Caricamento di %1$S su Photobucket in corso...
+photobucketservice.uploadProgress.title=Caricamento su Photobucket:
 
 #nsMsgFullTextIndexer.js
 #indexerProgress.js
 
 startIndexing=Inizia a indicizzare i messaggi...
 indexingMessageOnIdle=Indicizzazione in corso...
-calculatingMinFrecency=Calcolo in corso, attendere...
-cleaningUp=Pulizia in corso, attendere...
+calculatingMinFrecency=Calcolo in corso, attendi...
+cleaningUp=Pulizia in corso, attendi...
 
 #LOCALIZATION NOTE %1$S is the number of indexed messages, %2$S is the total number of message, %3$S is the subject of the message we indexed
-indexingMessageForced=Indexing message %1$S of %2$S:  '%3$S'
+indexingMessageForced=Indicizzazione messaggio %1$S di %2$S:  '%3$S'
 #LOCALIZATION NOTE %1$S is the number of indexed messages, %2$S is the total number of message, %3$S is the subject of the message we indexed
-downloadingMessageForced=Downloading message %1$S of %2$S:  '%3$S'
+downloadingMessageForced=Download messaggio %1$S di %2$S:  '%3$S'
 #LOCALIZATION NOTE %1$S is the folder pretty name, %2$S is the account pretty name
-preparingToIndex=Preparing to index '%1$S' on '%2$S'...
+preparingToIndex=Preparazione dell'indicizzazione di '%1$S' su '%2$S'...
 #LOCALIZATION NOTE %1$S is the folder pretty name, %2$S is the account pretty name
-searchingFolderOnAccount=Searching '%1$S' on '%2$S' for messages to index...
+searchingFolderOnAccount=Ricerca '%1$S' su '%2$S' di messaggi da indicizzare...
 #LOCALIZATION NOTE %1$S is the folder pretty name, %2$S is the account pretty name
-downloadingFolder=Downloading messages from '%1$S' on '%2$S'...
-preparingToIndexAll=Preparing to index all messages...
+downloadingFolder=Download messaggi da '%1$S' su '%2$S'...
+preparingToIndexAll=Preparazione dell'indicizzazione dei nuovi messaggi...
 #LOCALIZATION NOTE %1$S is the folder pretty name
-preparingToIndexInFolder=Preparing to index all messages in '%1$S'...
-#LOCALIZATION NOTE %1$S is count of messages indexed, %2$S is the folder pretty name, 
-indexingCompleteFolder=Done.  %1$S messages are indexed in '%2$S'.
+preparingToIndexInFolder=Preparazione dell'indicizzazione di tutti i messaggi in '%1$S'...
+#LOCALIZATION NOTE %1$S is count of messages indexed, %2$S is the folder pretty name,
+indexingCompleteFolder=Fatto. %1$S messaggi sono indicizzati in '%2$S'.
 #LOCALIZATION NOTE %1$S is count of messages indexed
-indexingComplete=Done.  %1$S messages are indexed.
+indexingComplete=Fatto. %1$S messaggi sono indicizzati.
 
 #MsgComposeCommands.js
-noSignature=No signature
+noSignature=Nessuna firma
 searching=Ricerca in corso…
 noSearchResults=Non ci sono elementi corrispondenti alla tua ricerca.
-relationshipType-syn=synonyms
-relationshipType-ant=antonyms
-relationshipType-rel=related terms
-relationshipType-sim=similar terms
-drivingDirections=Driving Directions
+relationshipType-syn=sinonimi
+relationshipType-ant=antonimi
```

Comment:
CORRECTION NEEDED. 

Though "antonimi" is formally correct, I would replace it with the more common "contrari".